### PR TITLE
[Full Screens - Auth] ResetPasswordScreen

### PR DESCRIPTION
**Pull request by manojleaton**
Fixes #BLUI-5122 .

#### Changes proposed in this Pull Request:

- ResetPasswordScreen component added.
- Added a new `accountParams` prop

#### Screenshots / Screen Recording (if applicable)
Screenshot 2024-04-11 at 11 54 08 PM

Screenshot 2024-04-15 at 4 34 34 PM
Screenshot 2024-04-15 at 4 37 41 PM
Screenshot 2024-04-15 at 4 38 20 PM
Screenshot 2024-04-15 at 4 46 44 PM

#### To Test:

- yarn start:example-ios/android
- Click on ResetPasswordScreen... button in navigation drawer and test

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/src/screens/ResetPasswordScreen/ResetPasswordScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+        SuccessScreenProps,
+    } = props;
+
+    const [passwordInput, setPasswordInput] = useState(PasswordProps?.initialNewPasswordValue ?? '');
+    const [confirmInput, setConfirmInput] = useState(PasswordProps?.initialConfirmPasswordValue ?? '');
+    const [hasVerifyCodeError, setHasVerifyCodeError] = useState(false);
+    const [isLoading, setIsLoading] = useState(false);
+    const [showSuccessScreen, setShowSuccessScreen] = useState(false);
+
+    // const { code, email } = parseQueryString(window.location.search);
```
Comment (ektaghag-eaton) [inline login-workflow/src/screens/ResetPasswordScreen/ResetPasswordScreen.tsx:45]: Please remove commented code.

File: login-workflow/src/screens/ResetPasswordScreen/ResetPasswordScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    const code = '';
```
Comment (surajeaton) [inline login-workflow/src/screens/ResetPasswordScreen/ResetPasswordScreen.tsx:46]: Here the routing part is moved to example and Reset password screen will be in the library. We may need to decide how to get code and email here. We can update this to use a prop similar to invite registration workflow.

cc @ektaghag-eaton @joebochill

File: login-workflow/src/screens/ResetPasswordScreen/ResetPasswordScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+    }, []);
+
+    const workflowCardBaseProps = {
+        loading: isLoading,
+        ...WorkflowCardBaseProps,
+    };
+
+    const workflowCardHeaderProps = {
+        title: t('bluiCommon:FORMS.RESET_PASSWORD'),
+        onIconPress: (): void => {
```
Comment (ektaghag-eaton) [inline login-workflow/src/screens/ResetPasswordScreen/ResetPasswordScreen.tsx:112]: we have reset data here as well

File: login-workflow/src/utils/parseQueryString.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+type SearchParams = {
+    [key: string]: string;
+};
+
+export const parseQueryString = (search: string): SearchParams => {
```
Comment (ektaghag-eaton) [inline login-workflow/src/utils/parseQueryString.tsx:5]: are we using parseQueryString anywhere ?

Comment (manojleaton) [reply login-workflow/src/utils/parseQueryString.tsx:5]: Removed

**Comment by ektaghag-eaton** [comment]
Working fine
Screenshot 2024-04-15 at 4 07 28 PM
Screenshot 2024-04-15 at 4 07 21 PM

Please remove commented code and merge this PR.

**Review by ektaghag-eaton — APPROVED** [review]

**Pull request merged**